llvm/lib/Target/AMDGPU/AMDGPUTargetMachine.cpp

```diff
@@ -434,6 +434,12 @@ static cl::opt<bool> EnableMaxIlpSchedStrategy(
     cl::desc("Enable scheduling strategy to maximize ILP for a single wave."),
     cl::Hidden, cl::init(false));
 
+static cl::opt<bool> EnableMaxMemoryClauseSchedStrategy(
+    "amdgpu-enable-max-memory-clause-scheduling-strategy",
+    cl::desc("Enable scheduling strategy to maximize memory clause for a "
+             "single wave."),
+    cl::Hidden, cl::init(false));
```

Comment:
If we're going to keep cl::opts for this, it should mirror the attribute. One enum cl::opt with the same options as the attribute